### DISCUSSION TITLE
What causes the "in process" event to end?

### DISCUSSION THREAD

**Discussion by jsbowling42** [blakeblackshear/frigate#5023] [Ask A Question] [answered]
i pulled into my driveway and parked in the garage, the car was still in the frame which is why i think it was still labeled "In Process" but my question is at point does that end? i feel like that event could last for hours with the car parked.
![inprocess](https://user-images.githubusercontent.com/35424582/211849669-d9637825-a48f-4662-8e91-480e0b652d8d.JPG)
![inprocessCarInframe](https://user-images.githubusercontent.com/35424582/211849680-22dfb744-5827-47e5-a613-8c2948100550.JPG)

**Comment by NickM-27** [comment, marked as answer]
It ends when the object is no longer visible. Events last from object visible to object gone. The event being hours long does not matter as long as your recording retention policy is setup the way you want.

**Reply by jsbowling42** [reply]
Sounds good, as far as getting noticed of that even before it ends. Is that doable? Would I just not wait for the “end” trigger type?

**Reply by NickM-27** [reply]
I mean yeah frigate sends many messages. My setup does a notification at the beginning of the event and then again at the end.
